Modules/faulthandler.c

```diff
@@ -1274,6 +1274,8 @@ PyExec_faulthandler(PyObject *module) {
 
 static PyModuleDef_Slot faulthandler_slots[] = {
     {Py_mod_exec, PyExec_faulthandler},
+    // XXX gh-103092: fix isolation.
+    //{Py_mod_multiple_interpreters, Py_MOD_PER_INTERPRETER_GIL_SUPPORTED},
```

Comment:
`Py_MOD_MULTIPLE_INTERPRETERS_NOT_SUPPORTED` here as well? It seems wrong to leave faulthandler with the default (support multi-phase init) until isolation is fixed.

Reply:
If we mark it as not-supported then when faulthandler is trigged by a crash (or whatever) in a subinterpreter, things can get very unhelpful quickly.  IIRC, the remaining isolation issues in the module involve unlikely races, so we should be okay with the default for now.

Reply:
Thanks for the explanation!